Assets/HotmaxEditorVR/init.cs

```diff
@@ -8,10 +8,20 @@ public class init : MonoBehaviour {
     //-- used for classes that frequently get destroyed but need reference to a gameObject
     public static GameObject rotationGizmos;
     public static GameObject props;
+    public static GameObject deletePanel;
+    public static GameObject deletedProps;
+    public static GameObject handColliderFollow;
+
+    public static stateManager _stateManagerMutatorRef;
 
     void Awake ()
     {
         rotationGizmos = GameObject.Find("rotationGizmos");
-        props = GameObject.Find("Props");
+        props = GameObject.Find("props");
+        deletePanel = GameObject.Find("deletePanel");
+        deletedProps = GameObject.Find("deletedProps");
+        handColliderFollow = GameObject.Find("handColliderFollow");
```

Comment:
Don't think you're using handColliderFollow anywhere

Reply:
deleted